### ISSUE TITLE
v10.6.4 breaks my app in several points

### ISSUE THREAD

**Issue by Pachat** [directus/directus#20110]
### Describe the Bug

- Some page will not load (timeout)
- Some accordion will not open
- Bug from strange generated SQL by Directus
`select "project"."id" from "project" inner join (select distinct "project"."id", "project"."id" as "sort_esoji" from "project" left join "project_network" as "ytntg" on "project"."id" = "ytntg"."project_id" left join "network" as "xakve" on "ytntg"."network_id" = "xakve"."id" where ("project"."id" = $1 and (("xakve"."network" = $2) or ("xakve"."network" = $3))) order by "project"."id" asc limit $4) as "inner" on "project"."id" = "inner"."id" order by "inner"."sort_esoji"` 
The bug says `invalid type for uuid`
Effectively `order by "project"."id"` is rather strange when the "id" is an uuid.
I searched on the whole database: no such query exists, neither in extensions.

Back to v10.2.1, everything works correctly.

### To Reproduce

Test thoroughly with Collections with Relations, Accordions, Use of uuid as id type, etc. 

### Directus Version

v10.6.4

### Hosting Strategy

Self-Hosted (Custom)

**Comment by paescuj** [comment]
Did you run the migrations [^1] and ensured no errors occurred during that?

---

> Some page will not load (timeout)

Randomly or always the same page?

> Some accordion will not open

Can you elaborate? How does that look like (maybe a screenshot)?

> Bug from strange generated SQL by Directus [...]

When does that error occur?

**Comment by Pachat** [comment]
I will try to give you as much information as possible.
But first: migrations
>Yes, I ran `npx directus database migrate:latest`
Is it enough when we come from v10.2.1 (15 versions before).
In other words: does `npx directus database migrate:latest` include all intermediates or only apply on the previous version.

**Comment by paescuj** [comment]
Yes, `npx directus database migrate:latest` is correct. It will apply all migrations since the previous version.

**Comment by Pachat** [comment]
Thank you for your answer.

I confirm the bugs 
On a bare new direct installation
1. Created a Test collection with an Input field within a Detail Group within an Accordion started close
2. => The Accordion will not open. Instead, the page stays busy.

Then I importing the DB that works under v10.2.1 into the v10.6.4 (and after migrate:latest) 
- Accordion will not open. Instead, the page is busy.
- I still get that weird query with  `invalid type for uuid` (which I don't get under v10.2.1)
- Opening/Closing successively a Detail Group (that this time is not inside an accordion) lead to refresh the page.
(That last point was not reproducible on my bare test).
